test/js/pageobjects/DocumentSelection.js

```diff
@@ -0,0 +1,11 @@
+import Base from './BasePage.js'
+
+class DocumentSelection extends Base{
+    getTitle() { return this.$('.onfido-sdk-ui-Title-titleSpan'); }
+    getPassport() { return this.$('.onfido-sdk-ui-DocumentSelector-icon-passport')}
+
+    copy = (lang="en") =>
```

Comment:
Is it used anywhere?

Reply:
not really to be honest. I think I did this in the webdriver io and added an assert then, but not on this one 🤔